datafusion/core/tests/sqllogictests/src/setup.rs

```diff
@@ -111,23 +111,23 @@ fn register_median_test_tables(ctx: &SessionContext) {
     }
 }
 
-async fn register_aggregate_csv_by_sql(ctx: &SessionContext) {
+pub async fn register_aggregate_csv_by_sql(ctx: &SessionContext) {
```

Comment:
I re-used the existing table creation function and updated the data types according to the sql from `integration-tests` https://github.com/apache/arrow-datafusion/blob/master/integration-tests/create_test_table.sql.

I can, of course, introduce another function.